### PR TITLE
added modules

### PR DESCRIPTION
**Pull request by fatmahussein**
- Re-created _Awesome Books app_  using ES6 syntax
- Created a separate file for each module in the modules directory
- Imported modules into the index.js file
- added Luxon library to get current date and time
- Kept all functionalities of the app without errors

### CODE REVIEW

File: books.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      removeBtn.textContent = 'Remove';
+      removeBtn.classList.add('remove');
+      removeCell.appendChild(removeBtn);
+      row.appendChild(removeCell);
+
+      booksList.appendChild(row);
+
+      removeBtn.addEventListener('click', () => this.removeBook(index));
+    });
+  }
```
Comment (frempongdev) [inline books.js]: Good going so far, displaying your books works as expected. 🚀I would, however, suggest that you consider using `arrow functions` instead as its introduction in `ES6` has many benefits like shorter syntax and implicit return statements, among others. ♻️ 

- [ ] Please convert all `ES5` functions to `ES6 arrow functions` as it is recommended. 🛸

File: date.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    default:
+      return 'th';
+  }
+}
+
+export function updateTime() {
+  const currentTimeElement = document.getElementById('current-time');
+  const currentTime = new Date();
+  currentTimeElement.textContent = formatDate(currentTime);
+}
```
Comment (frempongdev) [inline date.js]: Impressive implementation of `DateTime` in your `date.js` module 👍🏿  While everything works as expected, this milestone requires you to make use of '[Luxon](https://github.com/moment/luxon/blob/master/docs/install.md)' as a Date library to display the current date. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time) 📅 

- [ ] Kindly make sure the `Luxon` package is added as a `.js` file to the project (in full or minified version) and module is imported [ `import { DateTime } from "luxon";` ]. 🔽

**Review by frempongdev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @fatmahussein   👋🏼 
I'm Frempong.

You've done a great job implementing most of the requirements for this project 🚀, but there are a few changes I recommend be made. You're almost done, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ No Linter errors. 🆗  
✅ Books add and delete successfully


 ## Required Changes ♻️
Good job so far with this project. 🔥 However there is one requirement I would want to draw your attention to. 1️⃣ 

- [ ] Kindly make sure that you have a `modules` directory in your project setup with other `.js` files since the motive of this milestone is to learn the use of `modules`. 📁 

_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @fatmahussein, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Previous reviewer's request has been completed 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**